hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreFileInfo.java

```diff
@@ -51,6 +51,7 @@
 public class StoreFileInfo {
   private static final Logger LOG = LoggerFactory.getLogger(StoreFileInfo.class);
 
+  private FileStatus localStatus;
```

Comment:
Does it need to be volatile? Will StoreFileInfo be accessed by many threads? Can it be final/created on construction?